admin_guide/high_availability.adoc

```diff
@@ -14,34 +14,34 @@ toc::[]
 This topic describes how to set up highly-available services on your
 {product-title} cluster.
 
-The Kubernetes
-xref:../architecture/core_concepts/deployments.adoc#replication-controllers[replication
-controller] ensures that the deployment requirements, in particular the number
-of replicas, are satisfied when the appropriate resources are available. When
-run with two or more replicas, the
-xref:../architecture/core_concepts/routes.adoc#routers[router] can be resilient
-to failures, providing a highly-available service. Depending on how the router
-instances are discovered (via a service, DNS entry, or IP addresses), this could
-impose operational requirements to handle failure cases when one or more router
-instances are "unreachable".
+IP Failover uses link:http://www.keepalived.org/[*Keepalived*] to host a set of externally accessible
+Virtual IP (VIP) addresses on a set of hosts.  Each VIP is serviced by, at most, one of the hosts at
+any point in time.  *Keepalived* uses the VRRP protocol to determine which host in the set of hosts
+will service each VIP address.  If a host goes down or the service that it is watching goes down the
+VIP is serviced from another host in the set. As long as a single host is availble the VIP is serviced.
 
-For some IP-based traffic services, virtual IP addresses (VIPs) should always be
-serviced for as long as a single instance is available. This simplifies the
-operational overhead and handles failure cases gracefully.
+{product-title} supports an ipfailover DC created by the `oadm ipfailover` command. The ipfailover DC
+specifies the set of VIP addresses and the set of nodes on which to service them. The cluster can have
+multiple ipfailover DCs, each servicing its own set of VIP addresses. Each pod in the ipfailover DC runs
+*Keepalived* and each pod runs on a different node.
 
-IMPORTANT: Even though
-a service is highly available, performance can still be affected.
+When using VIPs to access a service, such as a router, the service should be running on all of the
+nodes that are running the ipfailover pods. This is done by using the same selector and replication count
+that is used by the ipfailover DC.
```

Comment:
@ramr in this case ipfailover will notice that one node doesn't have port 80 open and it won't participate in negotiation. So, yes it will work, but it seems like a mismatch in the configuration. I also used should, not must. What do you think? I will change the wording if you feel that this is important.